packages/cli/src/index.ts

```diff
@@ -152,39 +153,89 @@ cli.command('deploy', 'deploy the app to the marketplace')
         }
     });
 
-cli.command('create [appName]', 'create a new marketplace app').action(async (appName: string) => {
-    const { promptedAppName, stylingFramework } = await prompts([
-        {
-            type: 'text',
-            name: 'promptedAppName',
-            message: 'Enter your app name',
-            initial: appName || 'my-frontify-app',
-            validate: (value: string) => {
-                if (value.trim() === '') {
-                    return 'You need to enter an app name.';
-                }
-
-                return isValidName(value);
-            },
-        },
-        {
-            type: 'select',
-            name: 'stylingFramework',
-            message: 'Choose a styling framework',
-            choices: [
-                { title: 'Tailwind', value: 'tailwind' },
-                { title: 'CSS Modules', value: 'css-modules' },
-                { title: 'None', value: 'css' },
-            ],
-        },
-    ]);
-
-    if (!promptedAppName || !stylingFramework) {
-        exit(0);
-    }
+cli.command('create [appName]', 'create a new marketplace app')
+    .option('-e, --experimental', 'set experimental flag')
+    .action(async (appName: string, options) => {
+        if (options.experimental) {
+            const { promptedAppName, stylingFramework, appType } = await prompts([
+                {
+                    type: 'text',
+                    name: 'promptedAppName',
+                    message: 'Enter your app name',
+                    initial: appName || 'my-frontify-app',
+                    validate: (value: string) => {
+                        if (value.trim() === '') {
+                            return 'You need to enter an app name.';
+                        }
+
+                        return isValidName(value);
+                    },
+                },
+                {
+                    type: 'select',
+                    name: 'appType',
+                    message: 'Select the type of your app',
+                    choices: [
+                        { title: 'App', value: 'platformApp' },
+                        { title: 'Block', value: 'block' },
+                    ],
+                },
+                {
+                    type: (prev) => (prev === 'block' ? 'select' : null),
```

Comment:
Could you add the other "framworks"? It is just the matter of removing tailwindcss and postcss then we have parity (both CSS and CSS Modules are supported by default)

Reply:
Added, now we have them combined in the create method with the type as parameter. Should then also be fairly straight forward to add themes to it if needed.